docs/general/settings.rst

```diff
@@ -1,11 +1,163 @@
+
+.. confval:: DOWNLOAD_SEM_FACTOR
+
+    This factor determines the amount of simultaneous requests made towards the NIST API;
+    The set amount of client requests (30) get divided with the sem factor, so the lower
+    it is set, the more simultaneous requests are made.
+
+    If set, should be set ``>=0.6``.
+
+.. confval:: DOWNLOAD_SLEEP_MIN
+
+   Minimum time randomized sleep between (aiohttp) requests to NVD API.
+
+.. confval::  DOWNLOAD_SLEEP_MAX
+
+   Minimum time randomized sleep between (aiohttp) requests to NVD API.
+
+.. confval:: DOWNLOAD_BATCH_RANGE
+
+   Count of requests made in the time window of ``36`` seconds.
+
+   See https://nvd.nist.gov/general/news/API-Key-Announcement
+
+   Defaults to ``45`` if ``NVD_NIST_API_KEY`` is set, and to ``5`` without.
+
+Proxy
+*****
+
+A HTTP proxy can be used for database population & updates.
+
+As CveXplore is using both urllib3 and aiohttp for the connections, the proxy needs to be configured twice in forms supported by each.
+
+.. confval:: HTTP_PROXY_DICT
+
+   Dictionary of proxies used for HTTP & HTTPS connections.
+
+   This is used by urllib3 connections for both NVD API and other sources.
+
+   E.g., ``{ "http": "http://proxy.example.com:8080", "https": "http://proxy.example.com:8080" }``
+
+.. confval:: HTTP_PROXY_STRING
+
+   String presentation of the proxy.
+   
+   This is used by aiohttp for multiple asynchronous request to NVD API.
+
+   E.g., ``http://proxy.example.com:8080``
+
+Logging
+*******
+
+.. confval:: LOGGING_TO_FILE
+
+   Use file logging.
+
+.. confval:: LOGGING_FILE_PATH
+
+   Path for the log directory.
+
+.. confval:: LOGGING_MAX_FILE_SIZE
+
+   Maximum size for a log file.
+
+.. confval:: LOGGING_BACKLOG
+
+   How many files to keep at log rotate.
+
+.. confval:: LOGGING_FILE_NAME
+
+   Filename for log file.
+
+.. confval:: LOGGING_LEVEL
+
+   Short name of the maximum severity level of messages to be logged in log files.
+
+   ``DEBUG > INFO ( > NOTICE > WARNING > ERR > CRIT > ALERT > EMERG )``
+
+.. confval:: SYSLOG_ENABLE
+
+   Use syslog logging.
+
+.. confval:: SYSLOG_SERVER
+
+   IP address of the syslog server.
+
+.. confval:: SYSLOG_PORT
+
+   Port of the syslog server.
+
+.. confval:: SYSLOG_LEVEL
+
+   Short name of the maximum severity level of messages to be logged in syslog.
+
+   ``DEBUG > INFO ( > NOTICE > WARNING > ERR > CRIT > ALERT > EMERG )``
+
+   See https://www.rfc-editor.org/rfc/rfc5424.html#section-6.2.1
+
+.. confval:: GELF_SYSLOG
+
+    GELF format allows for additional fields to be submitted with each log record; Key values of this dict should
+    start with underscores; e.g. {"_environment": "SPECIAL"} would append an environment field with the value of
+    'SPECIAL' to each log record.
+
+.. confval:: GELF_SYSLOG_ADDITIONAL_FIELDS
+
+   See https://github.com/keeprocking/pygelf?tab=readme-ov-file#static-fields
+
+Redis
+*****
+
+.. confval:: REDIS_URL
+
+   Url to be used for contacting redis cache.
```

Comment:
While this section could go to the database section as Redis can be used as a database, for CveXplore Redis is more of a cache, and Redis related parameters are also documented as a part of the Celery backend documentation.

The main argument to have this here is that the 'common' settings for databases won't apply for Redis.